api/api/signals.py

```diff
@@ -70,42 +74,37 @@ async def check_installation_uid(app: web.Application) -> None:
             file.write(installation_uid)
             os.chown(file.name, common.wazuh_uid(), common.wazuh_gid())
             os.chmod(file.name, 0o660)
-    app[INSTALLATION_UID_KEY] = installation_uid
+    cti_context[INSTALLATION_UID_KEY] = installation_uid
 
 
 @cancel_signal_handler
-async def get_update_information(app: web.Application) -> None:
-    """Get updates information from Update Check Service and inject into the application context.
-
-    Parameters
-    ----------
-    app : web.Application
-        Application context to inject the update information.
-    """
+async def get_update_information() -> None:
+    """Get updates information from Update Check Service and inject into the global cti context."""
 
+    global cti_context
     while True:
         logger.info('Getting updates information...')
-        app[UPDATE_INFORMATION_KEY] = await query_update_check_service(app[INSTALLATION_UID_KEY])
-
+        cti_context[UPDATE_INFORMATION_KEY] = await query_update_check_service(cti_context[INSTALLATION_UID_KEY])
         await asyncio.sleep(ONE_DAY_SLEEP)
 
 
-async def register_background_tasks(app: web.Application) -> AsyncGenerator:
-    """Cleanup context to handle background tasks.
+@contextlib.asynccontextmanager
+async def lifespan_handler(_: ConnexionMiddleware):
+    """Logs the API startup/shutdown messages and register background tasks."""
 
-    Parameters
-    ----------
-    app : web.Application
-        Application context to pass to tasks.
-    """
     tasks: list[asyncio.Task] = []
 
     if running_in_master_node() and update_check_is_enabled():
-        tasks.append(asyncio.create_task(check_installation_uid(app)))
-        tasks.append(asyncio.create_task(get_update_information(app)))
+        tasks.append(asyncio.create_task(check_installation_uid()))
+        tasks.append(asyncio.create_task(get_update_information()))
+
+    # Log the initial server startup message.
+    start_stop_logger.info(f'Listening on {configuration.api_conf["host"]}:{configuration.api_conf["port"]}.')
```

Comment:
Shouldn't this message be logged with the `wazuh-apid` logger? Is there any reason why we would need to create a new one?